client/src/pages/Home/FindForm/FindForm.tsx

```diff
@@ -0,0 +1,94 @@
+import { useState } from 'react';
+import FormInput from '../../../components/FormInput/FormInput';
+import { Button, Grid, styled } from '@mui/material';
+import { LocalizationProvider } from '@mui/lab';
+import DateRangePicker, { DateRange } from '@mui/lab/DateRangePicker';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import useStyles from './useStyles';
+
+const SubmitBtn = styled(Button)(({ theme }) => ({
+  padding: theme.spacing(2.5, 5),
+}));
+
+const FindForm = () => {
+  const classes = useStyles();
+  const [dates, setDates] = useState<DateRange<Date>>([null, null]);
+
+  function handleSubmit() {
+    console.log('submitted');
+  }
+
+  return (
+    <LocalizationProvider dateAdapter={AdapterDateFns}>
```

Comment:
Maybe we can move the `LocalizationProvider` to one of the parents so that we can reuse it in another components